### ISSUE TITLE
Support for customizing letter-spacing / font tracking 

### ISSUE THREAD

**Issue by kunaaal13** [zed-industries/zed#16686]
### Summary

Settings to customize horizontal spacing between letters (font tracking)

<details><summary>Screenshot</summary>
Currently there is no setting to customise the spacing between lines in terminal and UI looks like this which is very cluttered.
<img width="1189" alt="Screenshot 2024-08-22 at 9 41 10 PM" src="https://github.com/user-attachments/assets/f2c302f9-660b-4ee8-be00-20cffdda5671">
</details>

**Comment by notpeter** [comment]
What you're looking for is something like:
```
  "terminal": {
     "line_height": {
       "custom": 1.5
     }
  },
```

It was documented in the default settings, but was missing from the web documentation:
https://github.com/zed-industries/zed/blob/89f14a3dfb6354ca1f7cc98d4175d00dfb9227f1/assets/settings/default.json#L668-L680

**Comment by kunaaal13** [comment]
what about letter spacing?

**Comment by fabianosl1** [comment]
any update about this?
